core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableSchema.java

```diff
@@ -255,21 +255,15 @@ public void readFields(DataInput in) throws IOException {
    * Below method will be used to build child schema object which will be stored in
    * parent table
    *
-   * @param className
-   * @param databaseName
-   * @param queryString
-   * @param queryType
-   *
-   * @return datamap schema
    */
-  public DataMapSchema buildChildSchema(String className, String databaseName, String queryString,
-      String queryType) {
+  public DataMapSchema buildChildSchema(String dataMapName, String className, String databaseName,
```

Comment:
Is the dataMapName is for child schema? Please modify the parameter name or describe in function comment in line 255